modules/raw-rsa-keyring-browser/src/raw_rsa_keyring_web_crypto.ts

```diff
@@ -193,6 +193,7 @@ export class RawRsaKeyringWebCrypto extends KeyringWebCrypto {
     const { wrappingAlgorithm, format, key } = getImportOptions(publicKey)
     const backend = await getWebCryptoBackend()
     const subtle = getNonZeroByteBackend(backend)
+    // @ts-ignore // TS2769: No overload matches this call
     return subtle.importKey(format, key, wrappingAlgorithm, false, ['wrapKey'])
```

Comment:
I am not smart enough to understand how `format` does not resolve to match an override.

I am open to suggestions to avoid `ts-ignore`.

Reply:
I tried to address with b7fbd96, but failed. WebCrypto's method overrides for `importKey` are below. 
`KeyFormat` there aliases to `"jwk" | "pkcs8" | "raw" | "spki"`. Clearly, the return of the ESDK's `getImportOptions` will always conform to a valid override of `importKey`. 

There is probably a way to convince TS that is the case. But I think the return annotation in b7fbd96 is enough of an effort for any developer to recognize that all is well. TS/JS will run the code happily; TS is just forcing us to be crystal clear. I suggest that we leave this as is.

```
importKey(format: "jwk", keyData: JsonWebKey, algorithm: AlgorithmIdentifier | RsaHashedImportParams | EcKeyImportParams | HmacImportParams | AesKeyAlgorithm, extractable: boolean, keyUsages: KeyUsage[]): Promise<CryptoKey>;
importKey(format: Exclude<KeyFormat, "jwk">, keyData: BufferSource, algorithm: AlgorithmIdentifier | RsaHashedImportParams | EcKeyImportParams | HmacImportParams | AesKeyAlgorithm, extractable: boolean, keyUsages: KeyUsage[]): Promise<CryptoKey>;
```